stellargraph/core/graph.py

```diff
@@ -203,8 +203,6 @@ class StellarGraph:
             Deprecated, use :meth:`from_networkx`.
         edge_type_name:
             Deprecated, use :meth:`from_networkx`.
```

Comment:
There's a couple of deprecated arguments still here. (+ `graph` and `node_type_name` above that github wont let me suggest changes for)

```suggestion
```

Reply:
I had carefully left these ones here despite removing the adjacent `edge_weight_label` because these arguments are not properly deprecated: they don't trigger a runtime warning yet. I think it's important that we use a runtime warning to prompt people to move because:

- user with existing code won't re-read the docs, and
- they may not realise that a breaking change listed in the CHANGELOG affects their code (as in, they may not remember all the details about their code)

These arguments don't trigger a runtime warning yet because "FIXME(#717): this should have a deprecation warning, once the tests and examples have stopped using it" on line 253/241 below.

Reply:
cool that makes sense; everything looks good then